### ISSUE TITLE
"Follow" buttons in tags search dropdown

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#6337]
Building on #6026

![image](https://user-images.githubusercontent.com/24359/65718069-c35b2480-e070-11e9-961f-a38cc29b0d75.png)

Let's implement this! I believe it's generated here: 
https://github.com/publiclab/plots2/blob/158594acacb46a5416c72215716abd6c1b60f52d/app/assets/javascripts/async_tag_subscriptions.js#L3-L17

However, when the item is selected, the updater is run, so we'd need to distinguish between clicks on the item vs. clicks on the `Follow` button. This may take some testing out.

**Comment by jywarren** [comment]
The new HTML code might be something like this:

```html
<li class="dropdown-item active">
  <a class="pull-right btn btn-sm btn-secondary" href="/subscribe/tag/balloon">Follow</a>
  <span href="#" role="option"></span>
</li>
```

It needs some tweaking. But as is, it doesn't allow clicking just on the button, so we'd need to work on that too. Maybe we could stop the `click` event from propagating up to the `li` somehow, so the button would just be a normal button and wouldn't trigger the `update()` function of the typeahead?

**Comment by CleverFool77** [comment]
Hi @sharmamansi You can look into this issue as you have already solved one of the first-timers only issue. 
Also, You can look for the issue with label `help-wanted`.
Thanks !!

**Comment by sharmamansi** [comment]
Hi , @CleverFool77  so can I start working on this ?

**Comment by jywarren** [comment]
Yes please! That would be great! Be sure to open a PR as soon as possible,
and share screenshots of your work! Thank you!!!

**Comment by sharmamansi** [comment]
Hi, @jywarren I'm getting trouble with proceeding in this issue whatever I thought would work is not correct i guess.  So can you please tell me how to start.

**Comment by jywarren** [comment]
Hi, can you open a pull request with what you've tried so far? Thanks!

**Comment by jywarren** [comment]
Hi, I wanted to say, i really tried on this but found it to be a total conundrum, and wasn't able to find any way in the docs of https://github.com/bassjobsen/Bootstrap-3-Typeahead/ to achieve this. So I'm going to leave it open but just a warning -- it's a really tough one!
